internal/handlers/handler_oauth2_token.go

```
@@ -79,3 +72,60 @@
 
 	ctx.Providers.OpenIDConnect.WriteAccessResponse(ctx, rw, requester, responder)
 }
+
+func handleOAuth2TokenHydration(ctx *middlewares.AutheliaCtx, rw http.ResponseWriter, req *http.Request, requester oauthelia2.AccessRequester, responder oauthelia2.AccessResponder, client oidc.Client, session *oidc.Session) (handled bool) {
```

golangci-lint (2.5.0)

[error] handleOAuth2TokenHydration - req is unused

(unparam)